official/projects/fffner/fffner_classifier.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""BERT cls-token classifier."""
```

Comment:
Also a general comment, I feel we used the functional API for the classifier and encoder. It might be better to use the subclassing way for better readability and maintenance.

Reply:
good point, I have updated the encoder part, let me know if that is what you want for the classifier as well.